athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/GlueConnectionUtils.java

```diff
@@ -43,6 +45,12 @@ public class GlueConnectionUtils
     private static final Logger logger = LoggerFactory.getLogger(GlueConnectionUtils.class);
     private static HashMap<String, HashMap<String, String>> connectionNameCache = new HashMap<>();
 
+    private static final ImmutableSet<String> GENERIC_JDBC_CONNECTORS = ImmutableSet.of(
```

Comment:
This shouldn't be in the federation SDK. 

The federation SDK should be completely source agnostic.